.eslintrc.js

```diff
@@ -5,13 +5,13 @@ module.exports = {
 
   extends: [
     'airbnb',
-    'plugin:flowtype/recommended',
+    // 'plugin:flowtype/recommended',
     'prettier',
-    'prettier/flowtype',
+    // 'prettier/flowtype',
     'prettier/react',
   ],
 
-  plugins: ['flowtype', 'prettier'],
+  plugins: ['prettier'],
```

Comment:
@jeroenransijn I tried a few solutions here but my linter kept yelling at me about `flowtype`.  I can remove this from the PR if you think you may implement flowtype in the future.